### PR TITLE
Add truncate to terminal.pyi

### PR DESCRIPTION
**Pull request by fishermans-friend**
In the recent PR: https://github.com/jquast/blessed/pull/212 the `Terminal.truncate()` method was added.

It was only added to the `sequence.pyi` file, not the `terminal.pyi` file, which may cause issues with autocompletion by IDEs.

I didn't bump the version number here, because as far as I know, 1.19.0 isn't actually released yet.

Another thing to consider is if maybe the width should be optional for this method, like it is for the `ljust()` and `rjust()` methods above it? It could truncate to terminal width by default, which might be nicer.

### CODE REVIEW

**Comment by avylove** [comment]
Makes sense to me to provide the default width. While the behavior of truncate is a little different, it would be consistent.

If you do your PRs from branches and leave master synced with upstream, it tends to be a bit cleaner.

**Comment by fishermans-friend** [comment]
Makes sense, I did think the history got a bit ugly. I'll try that next time.

I made width optional as well now.

File: tests/test_sequences.py [hunk 1]
```diff
@@ -716,3 +716,16 @@ def child(kind):
...
+
+def test_truncate_default(all_terms):
+    """Ensure that terminal.truncate functions with the default argument."""
+    @as_subprocess
+    def child(kind):
+        from blessed import Terminal
+        term = Terminal(kind)
+        test = "Testing " + term.red("attention ") + term.blue("please.")
+        trunc = term.truncate(test)
+        assert term.length(trunc) <= term.width
```
Comment (avylove) [inline tests/test_sequences.py:729]: i'd like to see a deterministic test,
Maybe `assert term.truncate(term.red(x * 1000)) == term.red(x * term.width)`

Comment (fishermans-friend) [reply tests/test_sequences.py:729]: Fixed in 6c06844

**Review by avylove — APPROVED** [review]

**Pull request merged**